lib/plenario_web/templates/web/chart/_info_block.html.eex

```diff
@@ -0,0 +1,68 @@
+<div class="panel panel-default">
```

Comment:
Panels don't exist in Bootstrap 4. If you like how this looks, you can just remove the `.panel` and `.panel-body` divs because they're not doing anything but being soup. If you actually want the visual effect, the new analogue is Cards.

(In the likely event you copied this from another view let me know and I'll go clean it up. I thought I got all the panels but I might have missed a view)